sdk/src/azure/core/az_http_policy.c

```diff
@@ -4,14 +4,14 @@
 #include "az_http_private.h"
 #include <azure/core/az_credentials.h>
 #include <azure/core/az_http.h>
+#include <azure/core/az_precondition.h>
 #include <azure/core/az_span.h>
+#include <azure/core/az_version.h>
 #include <azure/core/internal/az_http_internal.h>
 #include <azure/core/internal/az_result_internal.h>
 
 #include <azure/core/_az_cfg.h>
 
-static const az_span AZ_HTTP_HEADER_USER_AGENT = AZ_SPAN_LITERAL_FROM_STR("User-Agent");
```

Comment:
Why remove this, when we are still doing the User-Agent string creation, down below to call `az_http_request_append_header`?

Reply:
It is used only once, so I inlined it.